tests/ftqc/test_ftqc_graph_state_prep.py

```diff
@@ -50,7 +50,7 @@ def test_decompose(self):
         op = GraphStatePrep(qubit_graph=q)
         queue = op.decomposition()
         assert len(queue) == 20  # 8 ops for |0> -> |+> and 12 ops to entangle nearest qubits
-        for i in range(len(queue)):
+        for i in range(len(queue)):  # pylint: disable=consider-using-enumerate
             assert queue[i].name == "Hadamard" if i < len(lattice.nodes) else queue[i].name == "CZ"
```

Comment:
Can we follow the complaint and fix it, rather than ignore it?

```python
for idx,op in enumerate(queue)):
```
where `idx` is the numerical location, and `op` is the value at `queue[idx]` ?

Reply:
Done! Thank you!